config/initializers/permissions_policy.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+# Define an application-wide HTTP permissions policy. For further
+# information see https://developers.google.com/web/updates/2018/06/feature-policy
+#
+# Rails.application.config.permissions_policy do |f|
+#   f.camera      :none
```

Comment:
All the code has been commented out in here. Is this file needed?

Reply:
No, and I don't expect we'll need them in the future, but there's no harm in leaving them there in case. I think this now comes as part of the rails template